### PR TITLE
[REVIEW] assert if boolean else throw exception

### PR DESCRIPTION
**Pull request by rjd22**
@seggen-ibuildings & @arnovr

### CODE REVIEW

File: tests/LicenseManagerClientTest.php [hunk 1]
```diff
@@ -34,6 +34,24 @@ public function setUp()
     /**
      * @test
      */
+    public function shouldThrowExceptionWhenNotReceivingAValidBoolean()
+    {
+        $this->setExpectedException(
+            'DreamCommerce\LicenseManager\Exception\NotABooleanException',
+            'Expected a boolean got a string with value "\'true\'"'
```
Comment (seggen-ibuildings) [inline tests/LicenseManagerClientTest.php]: Maybe write a unit test for the NotABooleanException that checks the message, so you don't have to check for this message here?

Comment (seggen-ibuildings) [inline tests/LicenseManagerClientTest.php]: As discussed IRL... the message only makes sense in a context, so this is fine.

**Comment by seggen-ibuildings** [comment]
:+1:

**Comment by arnovr** [comment]
:+1:

**Pull request merged**